src/applications/disability-benefits/686c-674/actions/index.js

```diff
@@ -0,0 +1,39 @@
+// import recordEvent from 'platform/monitoring/record-event';
+// import { isServerError, isClientError } from '../config/utilities';
+
+export const VERIFY_VA_FILE_NUMBER_STARTED = 'VERIFY_VA_FILE_NUMBER_STARTED';
+export const VERIFY_VA_FILE_NUMBER_SUCCEEDED =
+  'VERIFY_VA_FILE_NUMBER_SUCCEEDED';
+export const VERIFY_VA_FILE_NUMBER_FAILED = 'VERIFY_VA_FILE_NUMBER_FAILED';
+
+// VA file number is required to create a valid entry in BGS.
+
+// This will eventually take apiRoute and options as it's params.
+// stub out a response for now.
+const getVaFileNumber = () =>
+  new Promise(resolve => {
+    setTimeout(() => {
+      resolve({ data: { attributes: true } });
+    }, 4000);
+  });
+
+// stub out error response
+// const getVaFileNumber = () =>
+//   new Promise(reject => {
+//     setTimeout(() => {
+//       reject({ errors: [{ code: 404, message: 'error' }] });
+//     }, 4000);
+//   });
+
```

Comment:
These are just stubs for now because the endpoint in vets-api has yet to be stood up. In a future ticket, these will be removed in place of an actual call to the backend.